modules/deploy/pages/deployment-option/cloud/api/cloud-api-overview.adoc

```diff
@@ -0,0 +1,158 @@
+= Redpanda Cloud API Overview
+:description: Learn about using the Cloud API to manage clusters and cloud resources.
+:page-cloud: true
+:page-api: cloud
+:page-api-slot: overview
+
+The Redpanda Cloud API is a collection of REST APIs that allow you to interact with different parts of Redpanda Cloud. You can call the API endpoints directly, or use tools like Terraform or Python scripts to automate cluster management.
+
+== Requirements
+
+To be able to use the Cloud API:
+
+* You must be a customer with an existing organization in Redpanda Cloud.
+* You can only use one organization for authentication. See xref:./cloud-api-authentication.adoc[] for steps to authenticate API requests.
```

Comment:
Please can we use full paths for xrefs? I know we've done it in the past but I think we should align on not doing so in future because it prevents single-sourcing from working